recipes/cogdl/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set name = "cogdl" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/THUDM/cogdl/archive/{{ version }}.tar.gz
+  sha256: 4c35bcc20e9bd7e9928c6e5a02d2ca974d42ad47255ad7dafd70cbec5da079d6
+
+build:
+  skip: true  # [win]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - gensim
+    - grave
+    - joblib
+    - matplotlib
+    - networkx
+    - numpy
+    - ogb
+    - optuna
+    - pandas
+    - prettytable ==0.7.2
+    - pytorch_cluster
+    - pytorch_geometric
+    - pytorch_scatter
+    - pytorch_sparse
+    - requests
+    - scikit-learn
+    - scipy
+    - six
+    - tabulate
+    - texttable
+    - tqdm
+
+test:
+  imports:
+    - cogdl
+    - cogdl.data
+    - cogdl.datasets
+    - cogdl.layers
+    - cogdl.models
+    - cogdl.models.emb
+    - cogdl.models.nn
+    - cogdl.tasks
+    - cogdl.trainers
+  requires:
+    - pytest
+  source_files:
+    - tests
+  commands:
+    # For sanity, run quick tests from a few different files
+    - pytest -k "test_gcn_cora or test_transe_fb13s or test_sortpool_mutag or test_stpgnn_infomax" tests
+
+about:
+  home: https://github.com/THUDM/cogdl
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'An Extensive Research Toolkit for Graphs'
+  description: |
+    CogDL is a graph representation learning toolkit that allows researchers
+    and developers to easily train and compare baseline or custom models for
+    node classification, link prediction and other tasks on graphs. It provides
+    implementations of many popular models, including non-GNN Baselines like
+    Deepwalk, LINE, NetMF, GNN Baselines like GCN, GAT, GraphSAGE.
+  doc_url: https://keg.cs.tsinghua.edu.cn/cogdl/
+  dev_url: https://github.com/THUDM/cogdl
+
+extra:
+  recipe-maintainers:
+    - eriknw
+    - paul-tqh-nguyen
```

Comment:
@paul-tqh-nguyen, are you okay with being a maintainer?

Reply:
Yes.